exp/anbox-cloud.md

```diff
@@ -98,11 +96,11 @@ For example, your web client can be a mobile app used to access the provided And
 <a name="lxd"></a>
 ## LXD
 
-Anbox Cloud includes LXD for hosting and managing the Ubuntu containers that run the nested Android containers.
+Anbox Cloud includes LXD for hosting and managing the Ubuntu instances that run the nested Android containers.
 
 LXD is installed through the [`ams-lxd` charm](https://charmhub.io/ams-lxd), which adds some Anbox-specific configuration to LXD. AMS configures LXD automatically and fully manages it, which means that in most scenarios, you do not need to worry about LXD at all.
 
-If you want to monitor LXD, you can always run `lxc list` to display the existing containers. For the full deployment, LXD hosts the AMS containers. If you run the Anbox Cloud Appliance, LXD also containerises the Anbox Cloud deployment, which means that it hosts containers for the different Juju machines that Anbox Cloud requires:
+If you want to monitor LXD, you can always run `lxc list` to display the existing instances. For the full deployment, LXD hosts the AMS instances. If you run the Anbox Cloud Appliance, LXD hosts instances for the different Juju machines that Anbox Cloud requires:
```

Comment:
After this change, it introduced a bit of confusion to me.
... For the full deployment, LXD hosts the AMS instances
->
I thought a dedicate instance to host AMS service however we're meant to say Anbox instances which have the leading `ams-` container name in such a case

Reply:
Discussed this in SU, split into a separate story.